Developers/Writing_Bears.rst

```diff
@@ -276,3 +276,98 @@ parameter in ``HiddenResult`` should be the instance of the Bear that yields
 this result (in this case ``self``), and second argument should be the content
 we want to transfer between the Bears. Here we use a list of strings as content
 but it can be any object.
+
+More Configuration Options
+--------------------------
+coala provides settings to further configure your bear according to your needs.
+Here are all the settings you can use:
+
+``LANGUAGES``: To indicate which languages your bear supports,
+just give it the ``LANGUAGES`` value which should be a set of string(s):
+
+.. code:: python
+
+    class SomeBear(Bear):
+        LANGUAGES = {'C', 'CPP','C#', 'D'}
+
+``REQUIREMENTS``: To indicate the requirements of the bear,
+assign ``REQUIREMENTS`` a set with instances of ``PackageRequirements``.
+
+.. code:: python
+
+    class SomeBear(Bear):
+        REQUIREMENTS = {
+        PackageRequirement('pip', 'coala_decorators', '0.2.1')}
+
+If your bear uses requirements from a manager we have a subclass from,
+you can use the subclass, such as ``PipRequirement``,
+without specifying manager:
+
+.. code:: python
+
+    class SomeBear(Bear):
+        REQUIREMENTS = {
+        PipRequirement('coala_decorators', '0.2.1')}
+
+To specify multiple requirements using ``pip``, you can use
+the multiple method. This can receive both tuples of strings,
+in case you want a specific version,or a simple string,
+in case you want the latest version to be specified.
+
+.. code:: python
+
+    class SomeBear(Bear):
+        REQUIREMENTS = PipRequirement.multiple(
+            ('colorama', '0.1'),
+            'coala_decorators')
+
+``INCLUDE_LOCAL_FILES``:  If your bear needs to include
+local files, then specify it giving strings containing relative
+file paths to the ``INCLUDE_LOCAL_FILES``
+
+.. code:: python
+
+    class SomeBear(Bear):
+        INCLUDE_LOCAL_FILES = {'checkstyle.jar',
+            'google_checks.xml'}
+
+``CAN_DETECT``, ``CANFIX``: To keep track easier of
+what a bear can do, simply tell it to the ``CAN_FIX`` and
+the ``CAN_DETECT`` sets.
+
+.. code:: python
+
+    class SomeBear(Bear):
+        CAN_DETECT = {'Syntax', 'Formatting', 'Security',
```

Comment:
first give an example, such as "CAN_DETECT = {'Syntax, Unused Code'}" etc and then a CAN_FIX with different values than CAN_DETECT, and afterwards: To view a full list of possible values, check this list: (and enumerate all the possible values here, those which you have given here already)